docs/governance.rst

```diff
@@ -0,0 +1,129 @@
+Binder Project Governance
+=========================
+
+The following sections describe governance and high-level principles
+surrounding the Binder Project.
+
+The Binder Project is a member of Project Jupyter, which is a fiscally
+sponsored project of NumFocus, a US 501c3 non-profit.
+
+Project mission
+~~~~~~~~~~~~~~~
+
+The mission of the Binder Project is to create, advance, and promote
+open technology that makes it easy for people to connect their data
+science communication, education, and scientific reproducibility work
+with computational environments which can run their work.
+
+The Binder Project pursues this mission by advancing the tools
+surrounding BinderHub, an open-source tool for hosting ad-hoc
+environments in the cloud. The project also serves as the primary
+maintainer of ``mybinder.org``, a public service and demonstration of
+the BinderHub technology.
+
+Membership
+~~~~~~~~~~
+
+Membership in the Binder Project is open to anyone who is nominated by
+an existing member. Members form the trusted group of individuals who
+manage the Binder Project.
+
+Membership comes with `privileges`_ and `responsibilities`_.
+
+Currently, the Binder Project team overlaps 100% with `the BinderHub
+team`_. However, these two groups may diverge in their membership over
+time.
+
+The authoritative list of members and their roles (alphabetical sorting):
+* Tim Head (red)
+* Min Ragan-Kelley (team leader)
+* Yuvi Panda (red)
+* Erik Sundell (blue)
+* Carol Willing (red)
+* Your Name (red or blue)
+
+Team privileges
+~~~~~~~~~~~~~~~
+
+-  recognition
+-  team members can speak for the project in public
+-  merge privileges on the project repositories
+-  red members can nominate other individuals to team membership
+
+Team responsibilities
+~~~~~~~~~~~~~~~~~~~~~
+
+Each member of the Binder Project team is expected to spend a
+"reasonable" (intentionally vague) amount of time contributing to the
+technology used by Project Binder as well as maintaining
+``mybinder.org``. This might mean contributing features, making patches,
+responding to issues, reviewing PRs, teaching about BinderHub, making
+organizational connections, welcoming new contributors, or evangelizing
+and helping others deploy BinderHub. They should be receptive to
+comments and ideas on the GitHub issues or in the Binder gitter channel.
+They should strive to make project decisions that reflect these
+community opinions.
+
```

Comment:
```suggestion

In addition, red team members are charged with growing
the Binder community and ensuring that we create an open and inclusive
culture for users and future team members.
```

Reply:
I didn't take over this suggestion as is. Instead I added 09325b4bd426e9687895bdf0d31b441978c1da16. What do you think of that?